src/Common/Crypto/KeyVersion.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/**
+ * KeyVersion enum
+ *
+ * Defines encryption algorithm versions used by OpenEMR's CryptoGen class.
+ * This versioning system allows for algorithm improvements while maintaining
+ * backwards compatibility for decrypting data encrypted with older versions.
+ *
+ * Version history:
+ * - Version 1: Basic AES-256-CBC with simple key derivation
+ * - Version 2-3: AES-256-CBC with HMAC-SHA256 authentication
+ * - Version 4-6: Modern AES-256-CBC with HMAC-SHA384 authentication and improved key derivation
+ *
+ * Storage differences:
+ * - Versions 1-4: Keys stored as base64 on drive (unencrypted)
+ * - Versions 5-6: Keys encrypted using database keys before drive storage
+ *
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ * @author    Michael A. Smith <michael@opencoreemr.com>
+ * @copyright Copyright (c) 2025 OpenCoreEMR Inc.
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Common\Crypto;
+
+enum KeyVersion: int
+{
+    case ONE = 1;
+    case TWO = 2;
+    case THREE = 3;
+    case FOUR = 4;
+    case FIVE = 5;
+    case SIX = 6;
+
+    /**
+     * Get the string representation of the key version
+     *
+     * @return string The string name (e.g., "one", "two", etc.)
+     */
+    public function toString(): string
```

Comment:
isn't this same as getting the `->value` from the enum?

Reply:
oh wait, I see it :)
(have int as the standard values and this gives the string)